react/README.md

```diff
@@ -279,6 +279,22 @@
   <div />
   ```
 
+- Avoid using array index as `key` prop. ([why?](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318))
```

Comment:
What do you think about phrasing this more generally, in a way that makes it clear what kinds of keys are good to use?